src/pudl/analysis/epacamd_eia.py

```diff
@@ -1,46 +1,18 @@
-"""Use the epacamd_eia crosswalk to connect EPA units to EIA generators and other data.
+"""Helper functions for filtering the EPA CAMD crosswalk table.
```

Comment:
This module now only contains the filtering... not sure if we should just rip it out altogether but it could certainly be helpful so I left it here and rejiggered the docs. Most of this module doc string I migrated over into the subplant_id metadata

Reply:
I think we can leave it here for now.